tests/test_data/eatthismuch.com/eatthismuch.json

```diff
@@ -0,0 +1,40 @@
+{
+  "author": "Eat This Much",
+  "canonical_url": "https://www.eatthismuch.com/calories/easy-garlic-chicken,33501",
+  "site_name": "Eat This Much",
+  "host": "eatthismuch.com",
+  "language": "en",
+  "title": "Easy Garlic Chicken",
+  "ingredients": [
+    "3 tbsp Butter",
+    "4 breast fillet Chicken breast",
+    "2 tsp Garlic powder",
+    "1 tsp Salt",
+    "1 tsp Onion powder"
+  ],
+  "instructions_list": [
+    "Melt butter in a large skillet over medium-high heat.",
+    "Add chicken to the skillet and sprinkle with garlic powder, salt, and onion powder.",
+    "Sauté for about 10 to 15 minutes per side, or until chicken is cooked through and juices run clear."
+  ],
+  "yields": "4 servings",
+  "total_time": 30,
+  "cook_time": 20,
+  "prep_time": 10,
+  "ratings": 5.0,
+  "nutrients": {
+    "servingSize": "529.2 grams",
+    "calories": "900.5925600000002",
+    "fatContent": "47",
+    "saturatedFatContent": "24.6",
+    "transFatContent": "1.4",
+    "carbohydrateContent": "6.4",
+    "sugarContent": "0.3",
+    "proteinContent": "107.9",
+    "sodiumContent": "2548",
+    "fiberContent": "1",
+    "cholesterolContent": "436.2",
```

Comment:
Some of these values seem non-ideal:

  * The decimal `calories` value suggests too much precision; on the recipe site it is displayed as an integer, but.. more generally I think there may be guidelines we could follow to determine a reasonable accuracy level to round/truncate to.
  * Some of the values lack units; on the one hand that might ensure that people check what those units are themselves; on the other hand some systems might incorrectly assume a default unit for certain fields.

Reply:
The European Commission provides some rounding rules that I believe could be implemented as a plugin: https://food.ec.europa.eu/document/download/3eb7952a-43b8-4c6a-8091-349ea707a9a7_en?filename=labelling_nutrition-vitamins_minerals-guidance_tolerances_1212_en.pdf#page=15

However: that's outside the scope of this pull request.  I'll note also that it requires knowledge of the measurement units referenced in some cases, in order to select the relevant logic rules to apply.

Because I'd prefer to figure out a way to harmonise the output values of the `nutrients` field, I think it makes sense for me to file a feature request for that (and perhaps to try implementing it in the nearish future).

Reply:
Filed as #1253.